locales/pt-br.lua

```diff
@@ -0,0 +1,280 @@
+        ["neutral"] = "Neutro",
+        ["neutral_desc"] = "Apenas um dia qualquer!",
+        ["smog"] = "Nevoeiro",
+        ["smog_desc"] = "Máquina de fumaça!",
+        ["foggy"] = "Neboluso",
+        ["foggy_desc"] = "Máquina de fumaça x2!",
+        ["overcast"] = "Nublado",
+        ["overcast_desc"] = "Não muito ensolarado!",
+        ["clouds"] = "Nuvens",
+        ["clouds_desc"] = "Cadê o sol?",
+        ["clearing"] = "Parcialmente nublado",
+        ["clearing_desc"] = "Nuvens começam a sair!",
+        ["rain"] = "Chuva",
+        ["rain_desc"] = "Faça chover!",
+        ["thunder"] = "Trovão",
+        ["thunder_desc"] = "Corra e se esconda!",
+        ["snow"] = "Neve",
+        ["snow_desc"] = "Está frio aqui fora?",
+        ["blizzard"] = "Nevasca",
+        ["blizzed_desc"] = "Máquina de neve?",
+        ["light_snow"] = "Pouca neve",
+        ["light_snow_desc"] = "Começando a sentir como o Natal!",
+        ["heavy_snow"] = "Neve pesada (NATAL)",
+        ["heavy_snow_desc"] = "Guerra de bolas de neve!",
+        ["halloween"] = "Halloween",
+        ["halloween_desc"] = "O que foi aquele barulho?!",
+        ["weather_changed"] = "Clima alterado para: %{value}",
+    },
+    commands = {
+        ["blips_for_player"] = "Mostrar blips para jogadores (somente administrador)",
+        ["player_name_overhead"] = "Mostrar nome do jogador na cabeça (somente administrador)",
+        ["coords_dev_command"] = "Ativar exibição de coordenadas para material de desenvolvimento (somente administrador)",
+        ["toogle_noclip"] = "Alternar noclip (somente administrador)",
+        ["save_vehicle_garage"] = "Salvar veículo em sua garagem (somente administrador)",
+        ["make_announcement"] = "Faça um anúncio (somente administrador)",
+        ["open_admin"] = "Abrir menu de administração (somente administrador)",
+        ["staffchat_message"] = "Envie uma mensagem para toda a Staff (somente administrador)",
+        ["nui_focus"] = "Dê NUI Focus a um jogador (somente administrador)",
+        ["warn_a_player"] = "Avisar um jogador (somente administrador)",
+        ["check_player_warning"] = "Verifique os avisos do jogador (somente administrador)",
+        ["delete_player_warning"] = "Excluir avisos de jogadores (somente administrador)",
+        ["reply_to_report"] = "Responder a um relatório (somente administrador)",
+        ["change_ped_model"] = "Alterar modelo de Ped (somente administrador)",
+        ["set_player_foot_speed"] = "Definir velocidade a pé do jogador (somente administrador)",
+        ["report_toggle"] = "Alternar relatórios de entrada (somente administrador)",
+        ["kick_all"] = "Expulsar todos os jogadores",
+        ["ammo_amount_set"] = "Defina sua quantidade de munição (somente administrador)",
+    }
+}
+
+if GetConvar('qb_locale', 'en') == 'pt-br' then
+    Lang = Locale:new({
+        phrases = Translations,
+        warnOnMissing = true,
+        fallbackLang = Lang,

```

Comment:
This line isn't needed just;

Lang = Locale:new({
    phrases = Translations,
    warnOnMissing = true
})